lib/llm-events/aws-bedrock/bedrock-command.js

```diff
@@ -37,7 +37,7 @@ class BedrockCommand {
       result = this.#body.max_tokens_to_sample
     } else if (this.isClaude3() === true || this.isCohere() === true) {
       result = this.#body.max_tokens
-    } else if (this.isLlama2() === true) {
+    } else if (this.isLlama() === true) {
```

Comment:
Is there a difference between the way 2 and 3 are invoked and processed? With Claude, the payloads for send and receive are completely different.

Reply:
👍